floss/language/rust/extract.py

```diff
@@ -17,12 +17,6 @@
 MIN_STR_LEN = 4
 
 
-def get_rdata_section(pe: pefile.PE) -> pefile.SectionStructure:
```

Comment:
references to this in rust/extract.py need to be updated

Reply:
added the function name in the import